src/system/core/config.js

```diff
@@ -150,6 +150,7 @@ FabulaUltima.i18n = {
 FabulaUltima.itemTypes = [
         "armor",
         "artifact",
+  "bond",
         "helper",
```

Comment:
```suggestion
        "artifact",
        "bond",
        "helper",
```

Reply:
I'm fine changing this to be in line. We should probably either keep tabs though or convert the entire codebase to spaces at some point